### PR TITLE
Minor description updates and added DeliveryRuleAction for origingrou…

### PR DESCRIPTION
**Pull request by nachakra**
…p override

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by erich-wang** [comment]
@nachakra , please fix the CI check errors, including ModelValidation, PrettierCheck and ModelValidation, thanks.

**Comment by nachakra** [comment]
@erich-wang - I fixed all errors. Please review again when you get a chance.

File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json [hunk 1]
```diff
@@ -3761,7 +3761,8 @@
             "ModifyRequestHeader",
             "ModifyResponseHeader",
             "UrlRedirect",
-            "UrlRewrite"
+            "UrlRewrite",
+            "DefaultOriginGroupOverride"
```
Comment (hytao) [inline specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json]: This is the action name expose to customer. Have PMs agreed on this name?

Comment (nachakra) [reply specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json]: Yes. Thanks

**Comment by nachakra** [comment]
@erich-wang  - Ping on this!

File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json [hunk 2]
```diff
@@ -3761,7 +3761,8 @@
             "ModifyRequestHeader",
             "ModifyResponseHeader",
             "UrlRedirect",
-            "UrlRewrite"
+            "UrlRewrite",
+            "OriginGroupOverride"
```
Comment (ryansbenson) [inline specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json]: Adding an enum value to a discriminator will introduce a breaking change as the SDK won't know how to serialize the new discriminator type.

Please see 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Comment (nachakra) [reply specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json]: We have not released this API version in prod as yet. And no SDK's have been generated for this version of the API as yet publicly.

**Comment by nachakra** [comment]
@ryansbenson - addressed your concern. Please take a look.

**Comment by nachakra** [comment]
@ryansbenson - Ping on this. Can I get a sign off on this please?

**Comment by ravbhatnagar** [comment]
SIgning off on this from ARM side. Please add modelasString:true to avoid breaking changes on SDK when added new values to an enum.

**Review by erich-wang — APPROVED** [review]

**Pull request merged**